wandb/sdk/wandb_init.py

```diff
@@ -286,6 +298,19 @@ def _log_setup(self, settings):
                 os.path.basename(settings.log_symlink_internal),
                 delete=True,
             )
+        else:
+            self._safe_copy(
+                os.path.dirname(settings.log_symlink_user),
+                settings.log_user,
+                os.path.basename(settings.log_symlink_user),
+                delete=True,
+            )
+            self._safe_copy(
+                os.path.dirname(settings.log_symlink_internal),
+                settings.log_internal,
+                os.path.basename(settings.log_symlink_internal),
+                delete=True,
+            )
```

Comment:
These arent needed, they are just convenience symlinks.  copy would be bad as they would copy a file that doesnt have any log data in them

Reply:
I see now, removed